MFVNeutralino/python/VertexSelector_cfi.py

```diff
@@ -103,33 +106,46 @@
 
 mfvSelectedVerticesLoose = mfvSelectedVertices.clone(
     mevent_src = 'mfvEvent',
+    exclude_beampipe = True,
     min_ntracks = 3,
-    min_bsbs2ddist = 0.005,
-    max_rescale_bs2derr = 0.05,
+    min_bsbs2ddist = 0.01,
+    max_rescale_bs2derr = 0.005,
     )
 
 mfvSelectedVerticesTight = mfvSelectedVertices.clone(
     mevent_src = 'mfvEvent',
     exclude_beampipe = True,
     min_ntracks = 5,
     min_bsbs2ddist = 0.01,
-    max_rescale_bs2derr = 0.0050,
+    max_rescale_bs2derr = 0.005,
     )
 
-mfvSelectedVerticesTightMinNtk3 = mfvSelectedVerticesTight.clone(min_ntracks = 3)
+mfvSelectedVerticesLoosetNtk5    = mfvSelectedVerticesLoose.clone() # for looping convenience
+mfvSelectedVerticesLooseNtk3 = mfvSelectedVerticesLoose.clone(min_ntracks = 3, max_ntracks = 3) 
+mfvSelectedVerticesLooseNtk4 = mfvSelectedVerticesLoose.clone(min_ntracks = 4, max_ntracks = 4)
+mfvSelectedVerticesLooseMinNtk3 = mfvSelectedVerticesLoose.clone(min_ntracks = 3) 
+mfvSelectedVerticesLooseMinNtk4 = mfvSelectedVerticesLoose.clone(min_ntracks = 4)
+
 mfvSelectedVerticesTightNtk3    = mfvSelectedVerticesTight.clone(min_ntracks = 3, max_ntracks = 3)
 mfvSelectedVerticesTightNtk4    = mfvSelectedVerticesTight.clone(min_ntracks = 4, max_ntracks = 4)
 mfvSelectedVerticesTightNtk3or4 = mfvSelectedVerticesTight.clone(min_ntracks = 3, max_ntracks = 4)
 mfvSelectedVerticesTightNtk3or5 = mfvSelectedVerticesTight.clone(min_ntracks = 3, max_ntracks = 5)
 mfvSelectedVerticesTightNtk4or5 = mfvSelectedVerticesTight.clone(min_ntracks = 4, max_ntracks = 5)
 mfvSelectedVerticesTightNtk5    = mfvSelectedVerticesTight.clone() # for looping convenience
 
+
+mfvSelectedVerticesLooseMinNtk3 = mfvSelectedVerticesLoose.clone(min_ntracks = 3)
+mfvSelectedVerticesLooseMinNtk4 = mfvSelectedVerticesLoose.clone(min_ntracks = 4)
+mfvSelectedVerticesLoosetNtk5    = mfvSelectedVerticesLoose.clone() # for looping convenience
+
 mfvSelectedVerticesSeq = cms.Sequence(
     mfvSelectedVerticesExtraLoose *
+    mfvSelectedVerticesLoose *
     mfvSelectedVerticesTight *
     mfvSelectedVerticesTightNtk3 *
     mfvSelectedVerticesTightNtk4 *
     mfvSelectedVerticesTightNtk3or4 *
     mfvSelectedVerticesTightNtk3or5 *
     mfvSelectedVerticesTightNtk4or5
     )
+
```

Comment:
With these definitions, is a loose vertex just inverting the track multiplicity? If so, all of the LooseNtk5, LooseNtk3, LooseNtk4, etc. seem unnecessary. We previously had much looser requirements, but very rarely used the loose vertices for much of anything...

I also don't know why we have mfvSelectedVerticesLoosetNtk5 and mfvSelectedVerticesLooseNtk (the first has a typo in the name), and similar for MinNtk3 and MinNtk4 appearing twice.